src/site/content/en/blog/introducing-model-viewer/index.md

```diff
@@ -25,33 +26,51 @@ available to everyone on the web. Unfortunately, it can be difficult to add a 3D
 model to a website in a way that provides a great user experience without a deep
 knowledge of 3D technologies or resorting to hosting 3D content on a third-party
 site. The `<model-viewer>` web component, [introduced in early
-2019](../model-viewer), seeks to
-make putting 3D models on the web as easy as writing a few lines of HTML. Since
-then, the team has been working to address feedback and requests from the
-community. The culmination of that work is `<model-viewer>` version 1.0.
+2019](../model-viewer), seeks to make putting 3D models on the web as easy as
+writing a few lines of HTML. Since then, the team has been working to address
+feedback and requests from the community. The culmination of that work was
+`<model-viewer>` version 1.0, relesed earlier this year. We're now announcing
+the release of `<model-viewer>` 1.1. You can [read the release
+notes](https://github.com/google/model-viewer/releases/tag/v1.1.0) in GitHub.
 
 ## What's new since last year?
 
-Version 1.0 includes built-in support for augmented reality (AR) on the web,
+Version 1.1 includes built-in support for augmented reality (AR) on the web,
 improvements to speed and fidelity, and other frequently-requested features.
 
 ### Augmented reality
 
 Viewing a 3D model on a blank canvas is great, but being able to view it in your
-space is even better. Now `<model-viewer>` supports [WebXR augmented
-reality](https://modelviewer.dev/examples/augmented-reality.html), for an
-entirely-within-the-browser 3D and AR experience on Chrome Android.
+space is even better. For an entirely-within-the-browser 3D and AR experience on
+Chrome Android the team is working support [augmented
```

Comment:
This should be reworded to say <model-viewer> supports AR using WebXR, correct?

Reply:
Fixed.